analyzer/src/funTest/kotlin/GradleTest.kt

```diff
@@ -141,13 +141,13 @@ class GradleTest : StringSpec() {
                     row("3.5", "-3.4"),
```

Comment:
Commit message: "suffix" instead of "postfix"?

Reply:
Done.